code/src/App.js

```diff
@@ -1,9 +1,82 @@
-import React from 'react'
+
+import React, { useState, useEffect } from 'react';
+
+import ThoughtForm from 'components/ThoughtForm'
+import ThoughtItem from 'components/ThoughtItem'
+
 
 export const App = () => {
+  const [thought, setThought] = useState([])
+  const [newThought, setNewThought] = useState('')
+  const [loading, setLoading] = useState(false)
+
```

Comment:
Would be nice with some comments in your code to better follow what you are doing!